bin/start-shuffle-server.sh

```diff
@@ -97,7 +100,7 @@ else
   exit 1
 fi
 
-$RUNNER $ARGS $JVM_ARGS $JAVA_LIB_PATH -cp $CLASSPATH $MAIN_CLASS --conf "$SHUFFLE_SERVER_CONF_FILE" $@ &> $OUT_PATH &
+$RUNNER $ARGS $JVM_ARGS $JAVA11_EXTRA_ARGS $JAVA_LIB_PATH -cp $CLASSPATH $MAIN_CLASS --conf "$SHUFFLE_SERVER_CONF_FILE" $@ &> $OUT_PATH &
```

Comment:
I've tried, the `-XX:+IgnoreUnrecognizedVMOptions` can be any where in the cmd line.
The order of options in cmd line do matter only in case of that there are  the exactly same options.